.github/workflows/test.yml

```
@@ -16,15 +20,35 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+env:
+  # If this is the original copy of PartCAD, then use the org's self-hosted runners
+  OSES: ["ubuntu-latest", "windows-latest", "macos-15-large", "macos-latest"]
```

actionlint (1.7.4)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)